openhcl/virt_mshv_vtl/src/processor/mshv/x64.rs

```diff
@@ -180,7 +180,6 @@ impl BackingPrivate for HypervisorBackedX86 {
         this: &'a mut UhProcessor<'p, Self>,
         vtl: GuestVtl,
     ) -> Self::StateAccess<'p, 'a> {
-        assert_eq!(vtl, GuestVtl::Vtl0);
```

Comment:
This should no longer be necessary after #125 gets merged i believe. We could wait, but i don't think it matters either way.